contracts/oss-diff.yml

```diff
@@ -2650,6 +2821,82 @@ components:
       required:
         - oldId
         - newId
+    RemoteConnection:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        orgId:
```

Comment:
Self-review: seems other APIs typically use `orgID`, would be good to be consistent